schema/patches/9.sql

```diff
@@ -1,3 +1,6 @@
+-- Link show tracklists to selector sources
+--  Moved from base.sql into a separate patch
+--  TODO Explain this better
```

Comment:
Used by BAPS and WebStudio for tracklisting (I think - @mstratford do you have the context for this?)

Reply:
I imagine it's just so you can arbitrarily link tracklist sources to selector sources but, not knowing how selector works in that way, it's hard to briefly describe it more than the db comment.